examples/with-remix-thirdpartyemailpassword/app/routes/_index.tsx

```diff
@@ -0,0 +1,110 @@
+import { CelebrateIcon, SeparatorLine, BlogsIcon, GuideIcon, SignOutIcon } from "../../assets/images";
+import { useSessionContext } from "supertokens-auth-react/recipe/session/index.js";
+import { signOut } from "supertokens-auth-react/recipe/thirdpartyemailpassword/index.js";
+import { recipeDetails } from "../config/frontend";
+import SuperTokens from "supertokens-auth-react";
+
+export default function Home() {
+  const session = useSessionContext();
+
+  if (session.loading) {
+    return <div>Loading...</div>;
+  }
+
+  if (session.doesSessionExist === false) {
+    return <div>Session does not exist</div>;
+  }
```

Comment:
this should not be required because this route is protected by SessionAuth

Reply:
Done.